app/upgrade_cudos.go

```diff
@@ -1960,6 +1965,27 @@ func checkDecTolerance(coins sdk.DecCoins, maxToleratedDiff sdk.Int) error {
 	return nil
 }
 
+func withdrawGenesisRemainingModulesBalance(genesisData *GenesisData, cudosCfg *CudosMergeConfig, manifest *UpgradeManifest) error {
+
+	if cudosCfg.Config.GenericModuleRemainingBalance == "" {
+		return fmt.Errorf("no remaining modules balances destination address provided")
+	}
+	for _, genesisAccountAddress := range genesisData.Accounts.Keys() {
+		genesisAccount := genesisData.Accounts.MustGet(genesisAccountAddress)
+		if genesisAccount.AccountType == ModuleAccountType && !genesisAccount.Balance.IsZero() {
+			memo := fmt.Sprintf("leftover_module_balance_%s", genesisAccount.Name)
+			err := moveGenesisBalance(genesisData, genesisAccountAddress, cudosCfg.Config.GenericModuleRemainingBalance, genesisAccount.Balance, memo, manifest, cudosCfg)
+			if err != nil {
+				return err
+			}
+
+		}
```

Comment:
Wouldn't it better to simply execut this fe lines in here https://github.com/fetchai/fetchd/blob/v0.14.0-rc14/app/upgrade_cudos.go#L2164-L2171?

We can drop whole this function, or?